musicrecs/templates/round/listen_phase.html

```diff
@@ -14,6 +14,10 @@
 
 {% block round_content %}
     <div class="bg-light" style="padding: 15px;">
+        <h4> Snoozin' recommendation type </h4>
+        <div class="row bg-dark text-light" style="padding: 10px; margin-top: 10px; margin-bottom: 10px; margin-left: 10px; margin-right: 10px; display: inline-block;">
+            <b> {{snoozin_rt}} </b>
+        </div>
```

Comment:
You should wrap the whole thing in a div that has margins so it doesn't blend into the music section below. Take a look at [the round description](https://github.com/nknaian/musicrecs/blob/master/musicrecs/templates/round/base.html#L21) as a reference

Reply:
As Jason mentioned, would probably be better if we stored the style stuff in a separate css file. That would make for less copy pasta

Reply:
Added the corrected approach for snoozin_rec_type to the round base file.

Reply:
So the only files that should have changed now are .../round/base.html and gitignore